assets/src/edit-story/panels/videoPoster.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import UploadButton from '../components/uploadButton';
+import { Panel, Title, getCommonValue } from './shared';
+
+const ButtonCSS = css`
```

Comment:
Shouldn't this be lowercase? It's not a component, just a variable.

Reply:
There variable is found in other parts of the codebase. Do you want me to change it everywhere?

Reply:
It really should be renamed, yes. `css` just [returns some structure](https://styled-components.com/docs/api#css) understood by `styled-components`, but not an actual component.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/135/commits/0231e59b88cdfbda73e0c5fb5b00d3b21f7e4b17